doc/help/MOUNT.md

```diff
@@ -0,0 +1,41 @@
+kdb-mount(1) - Mount a file to the key database
+===============================================
+
+## DESCRIPTION
+
+This command allows a user to mount a new "backend".  
+A backend allows a file to be interpreted as keys in the key database such that any edits to the keys are reflected in the file and vice versa.  
+Additionally, the user can use this command to list the currently mounted backends by running the command with no arguments.  
+
+## USAGE
+
+`kdb mount [<path> <mountpoint>] [<plugin> [<config>] [..]]`  
```

Comment:
@markus2330 I am playing around with the `-c` option and it seems like you would use it `kdb mount -c config=setting path mountpoint plugin1 plugin2...` is this how it works? Also can you show a real example of how the `-c` option is used? I just can't think of an example where someone should use this option?